openapi/openapi/src/main/scala/endpoints/openapi/JsonSchemas.scala

```diff
@@ -111,6 +111,8 @@ trait JsonSchemas extends endpoints.algebra.JsonSchemas {
 
   lazy val booleanJsonSchema: DocumentedJsonSchema = Primitive("boolean")
 
+  lazy val byteJsonSchema: DocumentedJsonSchema = Primitive("integer", format = Some("int32"))
```

Comment:
Unfortunately, `int32` is not appropriate for `Byte` and it seems that the OpenAPI specification does not have `int8`. Therefore, we should just use `None` for the `format` parameter.

```suggestion
  lazy val byteJsonSchema: DocumentedJsonSchema = Primitive("integer")
```